browser/src/Input/KeyBindings.ts

```diff
@@ -60,6 +62,8 @@ export const applyDefaultKeyBindings = (oni: Oni.Plugin.Api, config: Configurati
         input.bind("<m-r>", "browser.reload")
     } else {
         input.bind("<a-f4>", "oni.quit")
+        input.bind("<c-s-_>", "sidebar.decreaseWidth")
```

Comment:
Just a reminder that in many programs like terminals or other editors `<C-S-_,+>` is used to increase/decrease the font size, so I leave to you to ponder if we should adhere to this convention or not.

Reply:
@badosu I was thinking that control/cmd +/- was what most programs used to increased/decrease font size? if the convention is to use c-s-_/+ I can change these bindings to `c-+`/`c-_` ?

Reply:
@Akin909 In my keyboard the key `=` is the main one and `+` is the one in the shift position.

So, `<C-S-+>` would be not possible for me unless it was the same as `<C-S-=>`. Maybe I am misunderstanding anyway.

If someone else does not chime in with an insight please disregard my comment.

Reply:
I hit an unrelated issue with these keybinds.

On Windows, the library is resolving the keys as `<s-c-+>` and `<s-c-_>`, which means the default binds don't work. This is a flaw in the library (I think) and I found it again over in #1240.

Until thats sorted, its probably worth just swapping the bindings from `c-s-` to `s-c-` like the `commands.show` bind etc.

And @badosu you are correct that `=` is the default for that key, but since the bind has `shift` in it, it is resolved as `+`. So you are holding the `Ctrl`, `Shift` and `=/+` keys, but the library will resolve it to `c-s-+`.